Slashee/src/main/java/slashee/main/slack/model/SlackProfile.java

```diff
@@ -0,0 +1,127 @@
+package slashee.main.slack.model;
+
+import java.util.Map;
+
+import static com.google.common.base.Preconditions.checkNotNull;
+
+public class SlackProfile {
+
+    private static class SlackProfileKeys {
```

Comment:
Hmmmm, hardcoded ist nicht so toll - deshalb hatte ich ja über alle custom fields drüber iteriert. Besser wäre es, in der Config-Datei eine Liste mit Keys zu haben. Aber wenn das mit sehr viel Aufwand verbunden ist, müssen wir es vorerst so lassen & in einer der späteren Versionen fixen

Reply:
Das werde ich aktuell nicht ändern. Feel free das zu ändern, wenn du mal Zeit hast, aber mein Ansatz war make it work as quick as possible. ;)

Reply:
👍